code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -6400,4 +6400,24 @@ var/global/list/tonio_doesnt_remove=list("tonio", "blood")
 	if(holder.has_reagent(MUCUS))
 		holder.remove_reagent(MUCUS, 10)
 
+/datum/reagent/saltwater
+	name = "Salt Water"
+	id = SALTWATER
+	description = "It's water mixed with salt. It's probably not healthy to drink."
+	reagent_state = LIQUID
+	color = "#FFFFFF" //rgb: 255, 255, 255
+	density = 1.5
+	specheatcap = 4.184
+	
+/datum/reagent/saltwater/on_mob_life(var/mob/living/M)
+
+	if(..())
+		return 1
+
+	if(ishuman(M) && prob(20))
+		var/mob/living/carbon/human/H = M
+		H.vomit()
+		M.adjustToxLoss(4)
```

Comment:
Have you tested these numbers?

This is equally as toxic as bleach fumes, and more toxic than plasma.

Times it by REM for consistency